spec/controller/api/v3/device_registrations_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Api::V3::DeviceRegistrationsController, type: :controller do
+  describe "POST create" do
+    context "when registration id has already existed" do
+      let!(:device_registration) { FactoryGirl.create(:device_registration, registration_id: "some_example_registration_id") }
+      it "should not create a new device registration record" do
+        post :create, registration_id: "some_example_registration_id"
```

Comment:
origin_count = DeviceRegistration.count
post :create, registration_id: device_registration.registration_id
expect(DeviceRegistration.count).to eq(origin_count)